index.html

```diff
@@ -584,16 +584,27 @@ <h2><dfn>disconnect()</dfn> method</h2>
       </section>
       <section>
         <h2><a>supportedEntryTypes</a> attribute</h2>
-        <p>The user agent MUST maintain <dfn>supported entry types</dfn>, a list
-        of strings representing the entry types which the user agent supports
-        for the <a>PerformanceObserver</a> interface. This list is populated by
-        specifications that define new entry types via the
-        <a>register a performance entry type</a> algorithm.</p>
+        <p>Each <a data-cite="HTML/webappapis.html#global-object">global object</a> has
+          associated <dfn>supported entry types</dfn>, which is initially null but is later
+          initialized to be a <a data-cite=WEBIDL/#idl-frozen-array>FrozenArray</a> with
+          the set of strings representing the entry types which the user agent supports
+          for that global object, which must be a subset of those described in the
+          <a href="https://w3c.github.io/timing-entrytypes-registry/#registry">registry</a>.
+        </p>
         <p>When <dfn>supportedEntryTypes</dfn>'s attribute getter is called, run the following steps:</p>
         <ol>
-          <li>Let <var>result</var> be a copy of the user agent's <a>supported entry types</a>.</li>
-          <li>Sort <var>result</var> in alphabetical order.</li>
-          <li>Return <var>result</var>.</li>
+          <li>Let <var>globalObject</var> be the <a href=
+            "https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-global">
```

Comment:
You should probably use the "relevant global object of the context object" instead?

Reply:
Also, you should use `data-cite` rather than `href`

Reply:
There is no context object here, supportedEntryTypes is static. Changed to data-cite.